src/Build.UnitTests/BackEnd/MockLoggingService.cs

```diff
@@ -300,6 +300,17 @@ public void LogCommentFromText(BuildEventContext buildEventContext, MessageImpor
             _writer(message);
         }
 
+        /// <inheritdoc />
+        public void LogCommentFromText(BuildEventContext buildEventContext, MessageImportance importance, string message, params object[] messageArgs)
```

Comment:
I wonder what the gains of offering a few overloads for it would be:


```csharp
public void LogCommentFromText(BuildEventContext buildEventContext, MessageImportance importance, string message, params object[] messageArgs)
public void LogCommentFromText(BuildEventContext buildEventContext, MessageImportance importance, string message, object arg1)
public void LogCommentFromText(BuildEventContext buildEventContext, MessageImportance importance, string message, object arg1, object arg2)
```

That would avoid most of the allocations for the string.Format calls, I guess.

Reply:
We’ll need an array later on anyway, to store the arguments on LazyFormattedBuildEventArgs